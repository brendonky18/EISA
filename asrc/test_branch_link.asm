MOV R0, 10
BL #3
ADD R0, R0, 5
MULT R0, R0, 2
STR R0, #6
END