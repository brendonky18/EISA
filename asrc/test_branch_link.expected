00000100000000001000000000001010
01111111100000001000000000000011
00000100000000001000000000000101
00010000000000001000000000000010
00111000000000001000000000000110
10000000000000000000000000000000